smoke-tests/src/test/java/com/splunk/opentelemetry/AppServerTest.java

```diff
@@ -100,11 +106,15 @@ private String tryGetResponse(Request request) throws IOException {
     return responseBody;
   }
 
-  protected void additionalWebAppTraceAssertions(
-      TraceInspector traces, ExpectedServerAttributes serverAttributes) {}
+  protected void additionalWebAppTraceAssertions(TraceInspector traces, TestImage testImage) {}
 
-  protected int totalNumberOfSpansInWebappTrace() {
-    return 3;
+  protected int totalNumberOfSpansInWebappTrace(
```

Comment:
This is never overwritten, so can be private. Also `serverAttributes` is not used.